http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/sysdeps/unix/sysv/linux/sys/epoll.h.diff?r1=1.2&r2=1.3&cvsroot=glibc
Include <stdint.h>.

Fixes error
/foo/gcc-3.3.2-glibc-2.3.2/powerpc-750-linux-gnu/sys-include/sys/epoll.h:60: error: parse error before "uint32_t"
/foo/gcc-3.3.2-glibc-2.3.2/powerpc-750-linux-gnu/sys-include/sys/epoll.h:66: error: parse error before "uint32_t"

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/sys/epoll.h,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/sysdeps/unix/sysv/linux/sys/epoll.h	2003/03/24 23:50:25	1.2
+++ libc/sysdeps/unix/sysv/linux/sys/epoll.h	2003/06/13 19:49:50	1.3
@@ -19,6 +19,7 @@
 #ifndef	_SYS_EPOLL_H
 #define	_SYS_EPOLL_H	1
 
+#include <stdint.h>
 #include <sys/types.h>
 
 
